src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -113,65 +113,65 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
     fluid.defaults("fluid.slider", {
```

Comment:
The "default" should be the native HTML5 widget. However, we can apply this default in a cool way, by using the "makeGradeLinkage" facility targetted at the base grade - http://docs.fluidproject.org/infusion/development/IoCAPI.html#fluid-makegradelinkage-linkagename-inputnames-outputnames- - in this way, the default can be easily overridden on a global or local basis.